src/components/logo.tsx

```diff
@@ -54,16 +71,16 @@ export default class InteractiveLogo extends React.Component<LogoProps> {
         contextMenu: "off",
         splashScreen: false,
         preferredRenderer: "canvas",
+      }).catch(() => {
+        this.createStaticLogo();
       });
       this.player.style.width = "100%";
       this.player.style.height = "100%";
+    } else {
+      this.createStaticLogo();
     }
   }
 
-  componentDidMount() {
-    this.load();
```

Comment:
This seemed unnecessary with the `onReady`, and was causing `load` to always run once without `this.player`, messing with the code I added.